kiwi/package_manager/apt.py

```diff
@@ -513,3 +501,54 @@ def _package_requests(self) -> List:
         items = self.package_requests[:]
         self.cleanup_requests()
         return items
+
+    def _run_bootstrap_scripts(
+        self, solved_debootstrap_packages: List[str], only_for: List[str] = []
+    ):
+        # FIXME: weird stuff
+        # These env variables are required for the package scripts
+        # to work. I don't understand their meaning and I'm not sure
+        # if hardcoding this for libc6 is something we can afford ?
+        # Help is appreciated
+        self.command_env['DPKG_MAINTSCRIPT_NAME'] = 'true'
+        self.command_env['DPKG_MAINTSCRIPT_PACKAGE'] = 'libc6'
+
+        post_script_dir = Temporary(
+            prefix='kiwi_debpost.', path=self.root_dir
+        ).new_dir()
+        for package in solved_debootstrap_packages:
+            package_base_name = os.path.basename(package)
+            go_ahead = True
+            if only_for:
+                for name in only_for:
+                    if name not in package_base_name:
+                        go_ahead = False
+                        break
+            if not go_ahead:
+                continue
+            log.debug(
+                f'Running pre/post scripts for: {package_base_name}'
+            )
+            package_metadata_dir = \
+                f'{post_script_dir.name}/{os.path.basename(package)}'
+            Command.run(
+                ['dpkg', '-e', package, package_metadata_dir]
+            )
+            script_pre = f'{package_metadata_dir}/preinst'
+            script_post = f'{package_metadata_dir}/postinst'
+            # 1. preinst
+            if os.path.exists(script_pre):
+                Command.run(
+                    [
+                        'chroot', self.root_dir, 'bash',
+                        f'{script_pre.replace(self.root_dir, "")}', 'install'
+                    ], self.command_env
+                )
+            # 2. postinst
+            if os.path.exists(script_post):
+                Command.run(
+                    [
+                        'chroot', self.root_dir, 'bash',
+                        f'{script_post.replace(self.root_dir, "")}', 'configure'
+                    ], self.command_env
+                )
```

Comment:
@IsaacJT If you know a shorter or more straight forward way to call them all I appreciate your feedback.
In any case I had to run the `base-passwd` script first because that creates users, e.g root which is required for any other package script to work. In the apt sequence though it was not first and thus I had this strange two pass system where I run package scripts in a manual order first and then I do it again with the order of the apt resolver. It all feels not really perfect to be honest :)